{"name": "test_trigger_test_button", "status": "failed", "statusDetails": {"message": "AssertionError: Failed\nassert 1 == 2", "trace": "def test_trigger_test_button():\n        print(\"Testing Trigger Test Button\")\n    \n>       assert 1==2, \"Failed\"\nE       AssertionError: Failed\nE       assert 1 == 2\n\ntests\\test_stream.py:6: AssertionError"}, "attachments": [{"name": "stdout", "source": "8e2b7ff4-1108-4226-ab40-d021a05b04ec-attachment.txt", "type": "text/plain"}], "start": 1740428233718, "stop": 1740428233722, "uuid": "1e80c953-3831-4d89-9149-0a2261e0a945", "historyId": "1f8fbfd19b9570e4ed564d5bd541a9d6", "testCaseId": "1f8fbfd19b9570e4ed564d5bd541a9d6", "fullName": "tests.test_stream#test_trigger_test_button", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_stream"}, {"name": "host", "value": "DESKTOP-LUCDFB2"}, {"name": "thread", "value": "29836-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_stream"}]}